third_party/amd/lib/TritonAMDGPUToLLVM/TargetInfo.cpp

```diff
@@ -99,6 +99,10 @@ bool TargetInfo::canUseStMatrix(RankedTensorType tensorTy,
   return false;
 }
 
+bool TargetInfo::canUseLDSTransLoad(int bitwidth) const {
+  return arch == "gfx950" && llvm::is_contained({16, 8, 4, 6}, bitwidth);
```

Comment:
Since this PR only enabled bitwidth = 16, we need to respect the limitation here. And add a TODO saying that we will support other bitwidth in future PRs.